docs/current/sdk/go/guides/110632-embed-directories.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: /110632/embed-directories
+---
+
+# Copy embedded directories into a container
+
+Dagger does not expose the option to copy entire directories as a single step (yet). It is, however, doable by walking over it.
```

Comment:
Is it the option to copy entire directories from local host to a container or it also concerns copying directory from a container to another container?

Reply:
Second option